spec/features/user_visits_book_show_page_spec.rb

```diff
@@ -0,0 +1,35 @@
+require 'rails_helper'
+
+describe "User visits book show page" do
+    before :each do
+        @book = Book.create(title: "Game of Thrones")
+        @review1 = @book.reviews.create(user: "Erika", body: "Great book!", rating: 4)
+        @review2 = @book.reviews.create(user: "Allison", body: "Ok book", rating: 2)
+
+        visit book_path(@book)
+    end
+    
+    it "I see page with attributes and ratings" do
+        expect(page).to have_content("#{@book.title}")
```

Comment:
No need to use string interpolation here. RSpec will expect the page to have whatever `@book.title` evaluates to, which is a string.

Would expect to use interpolation if you were using a more specific expectation, e.g. `"Title: #{@book.title}"`.